app/react/sidebar/Sidebar.module.css

```diff
@@ -45,3 +46,19 @@
   margin: 0;
   list-style: none;
 }
+
+/* make the scrollbar track background transparent */
+.nav-list-container::-webkit-scrollbar {
```

Comment:
this is not supported by all the browsers. https://developer.mozilla.org/en-US/docs/Web/CSS/::-webkit-scrollbar

Reply:
Yeah it's still a big style improvement for the webkit browsers. For the unsupported firefox browsers there is a small improvement, giving the scrollbar a darker default bg with  color-scheme: dark . I'll make a comment for testers to have a look on firefox to make sure that there is no issue introduced.